pkg/types/aws/availabilityzones.go

```diff
@@ -5,4 +5,37 @@ const (
 	AvailabilityZoneType = "availability-zone"
 	// LocalZoneType is the type of Local zone placed on the metropolitan areas.
 	LocalZoneType = "local-zone"
+	// ZoneOptInStatusOptedIn is the opt-in status of the zone.
+	// For Availability Zones, this parameter always has the value of opt-in-not-required.
+	// For Local Zones and Wavelength Zones, this parameter is the opt-in status.
+	ZoneOptInStatusOptedIn = "opted-in"
 )
+
+// Zones stores the map of Zone attributes indexed by Zone Name.
+type Zones map[string]*Zone
+
+// Zone stores the Availability or Local Zone attributes used to set machine attributes, and to
+// feed VPC resources as a source for for terraform variables.
+type Zone struct {
```

Comment:
I defined those in `pkg/types/aws` because it is used in these packages (not sure if it wasn't a good approach =] ):
- `pkg/asset/installconfig/aws`
- `pkg/asset/machines`
- `pkg/tfvars`

Reply:
thanks for the explanation in Slack. Changes applied